docs/README.md

```diff
@@ -44,3 +45,26 @@ We recommend the following common development tools:
 * [Mailhog](https://github.com/Chassis/MailHog) - Captures outbound email from Altis and provides a fake inbox
 
 Consult the [Chassis documentation](http://docs.chassis.io/en/latest/extend/) for information about installing additional extensions.
+
+
+## Using HTTPS locally
+
+Local Chassis will generate an HTTPS security certificate you can use to run your local environment over HTTPS. The file will be located in the `/chassis` directory, by default it will be called `altis.local.cert` but if you have customised the `hosts` in `config.local.yaml` it will use the first host name in that list for the file name.
```

Comment:
Just wanted to make sure that this cert actually will work for all the hostsnames you add to your config? Currently we have sites that use a chassis config with subdomains too.

Reply:
Goood question. It did but looking at this now https://github.com/Chassis/chassis-openssl/blob/master/modules/openssl-nginx/templates/cert.cnf.erb#L41

Seems like that might be a bug. 🕷

Reply:
So not a bug, it references the `[ alt_names ]` section of the cnf file, however we weren't generating DNS entries for all items under the hosts key, fixed in https://github.com/Chassis/chassis-openssl/pull/3